packages/editor/modules.d.ts

```diff
@@ -2,3 +2,8 @@ declare module '*.png' {
 	const url: string
 	export default url
 }
+
+declare module '*.svg' {
```

Comment:
I think we could remove the declare module for the png above, I only added it for the temporary watermark.